### PR TITLE
Fix RequestAssets crash with non-ETH default account

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Review by vilsbole — CHANGES_REQUESTED** [review]
When clicking on the `Request` link in the nav menu, the app throws the following error: 

Screenshot 2021-01-25 at 14 21 49

File: src/utils/wallets.ts [hunk 1]
```diff
@@ -7,12 +7,12 @@ export const getAccountsByNetwork = ({
...
 }: {
   accounts: StoreAccount[];
-  networkId: NetworkId;
+  networkId?: NetworkId;
   includeViewOnly?: boolean;
 }) =>
   accounts.filter(
     (acc) =>
-      acc.networkId === networkId &&
+      (acc.networkId === networkId || !networkId) &&
```
Comment (vilsbole) [inline src/utils/wallets.ts:15]: I looks like monkey patching.

**Review by vilsbole — CHANGES_REQUESTED** [review]

**Comment by FrederikBolding** [comment]
Can't reproduce your error, this is the error this should fix. @vilsbole

File: src/utils/wallets.ts (hunk 1 above)
Comment (FrederikBolding) [reply src/utils/wallets.ts:15]: If you have suggestions I am open to it.

**Review by vilsbole — APPROVED** [review]

**Pull request merged**